src/test/java/org/jenkinsci/plugins/credentialsbinding/masking/BashPatternMaskerProvider2Test.java

```diff
@@ -52,6 +54,7 @@
     @Before
     public void setUp() {
         assumeThat("bash", is(executable()));
+        j.jenkins.getDescriptorByType(Shell.DescriptorImpl.class).setShell(Executables.getPathToExecutable("bash"));
```

Comment:
Due to that, you remove the test for "pure Shell" right? But we want to support it along Bash, Batch and PowerShell.